xml/System.Buffers/ArrayPool`1.xml

```diff
@@ -34,8 +42,8 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Invokes the static constructor when a new instance of the <see cref="ArrayPool{T}"/> class is instantiated.</summary>
```

Comment:
Just document this as "Initializes a new instance of the ArrayPool<T> class."
You can also add a Remarks section that reads, "Note that this constructor is protected; it can only be called by classed derived from the ArrayPool<T> class."